packages/modular-scripts/src/cli.ts

```diff
@@ -137,6 +137,15 @@ program
     return start(packageName);
   });
 
+program
```

Comment:
Should this new command be added to the README?

Reply:
Good shout - added something for the command.